gateway.js

```diff
@@ -0,0 +1,182 @@
+/*
+ * Gulp Common - Microsoft Sample Code - Copyright (c) 2016 - Licensed MIT
+ */
+'use strict';
+
+var fs = require('fs');
+var args = require('get-gulp-args')();
+
+var all;
+
+function initTasks(gulp, options) {
+  all = require('./all.js')(options);
+
+  var config = all.getConfig();
+  var workspace = './ble_sample/';
+
+  // stick config into gulp object
```

Comment:
that's the same as in other gulf files, this is the simplest way to access config externally.